Equilobe.DailyReport.SL/DbExtensions/AtlassianUserExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Equilobe.DailyReport.DAL;
+using Equilobe.DailyReport.Models.Storage;
+using System.Linq;
+
+namespace Equilobe.DailyReport.SL.DbExtensions
+{
+    public static class AtlassianUserExtensions
+    {
+        public static IQueryable<AtlassianUser> GetAtlassianUsers(this ReportsDb db, long instanceId, bool? isActive = null, bool? isStalling = null)
+        {
+            return db.AtlassianUsers
```

Comment:
This methods should be moved inside a service class. I said earlier that we should create data service classes and do the db calls there. Not extension methods

Reply:
I just tried, and I can't have IQueryable as a return type, if I close the using of the DbContext. And as I said, there are a lot of cases in which I retrieve data from db, only to update it.